Add KCD Mexico 2025 event (#62)
opensource-latinamerica/openinframx: content/events/kcd-mexico2025/index.md

---
title: KCD Mexico in Guadalajara, Mexico 2025.
date: 2025-05-07T19:12:00+08:00
featured: true
draft: false
comment: false
toc: true
reward: false
pinned: false
carousel: true
series:
  - News
categories:
  - Events
tags: 
  - Advertising
authors:
  - rodolfo.martinez
images: ["/images/events/kcd_mexico_2025.jpg"]
---

KCD Mexico in Guadalajara, Mexico 2025

<!--more-->

On Saturday, March 29, Guadalajara transformed into the hub of cloud-native technology with the grand
celebration of Kubernetes Community Day 2025.
This remarkable event gathered over 200 professionals, enthusiasts, and experts in Kubernetes and cloud
computing, creating an atmosphere of innovation and collaboration.

<!-- image attendees -->

## A Day of Learning and Connection

> The event was a vibrant platform where the community exchanged ideas, tools, and best practices that are
> shaping the present and future of infrastructure as code. With a focus on technical and collaborative
> approaches, the day was filled with opportunities for learning, connecting, and building together.

Reviewer: ```suggestion
The event provided a vibrant platform where the community exchanged ideas, tools, and best practices that
shapes the future of infrastructure as code. With a depth focus on technical content and collaborative
environment, the day was packed with opportunities for learning, connecting, and building together.
```

## A Global Agenda

Featuring more than 30 international speakers from the US, Spain, Costa Rica, Colombia, Brazil, and Guatemala,
alongside national leaders, the event offered a diverse range of talks, workshops, and live demos.
Key topics included optimization, security, and scalability in Kubernetes, with a focus on tools and strategies
to maximize the potential of cloud-native environments.

## Highlights and Key Topics

The event delved into crucial topics related to Kubernetes optimization and efficiency, emphasizing tools that
enhance resource management, security, and application scalability. Highlights included:
- Karpenter + Amazon EKS: Intelligent autoscaling with Spot instances and Graviton processors for optimal performance and cost efficiency.
- Security First: Hardening strategies to comply with regulations like PCI DSS, HIPAA, and GDPR, fortifying clusters against real threats.
- API Gateway, OpenTelemetry, and Cilium: Enhanced visibility, traceability, and control in distributed environments.
- Istio + KubeVirt: Hybrid architectures combining the best of containers and virtual machines.

> - Cloud-native CI/CD: Migrations from GitHub Actions to solutions like Tekton and KubeVirt.

Reviewer: ```suggestion
- Cloud-native CI/CD: Migrations from GitHub-Hosted Actions to Self-Hosted solutions with KubeVirt.
```

- Green Infrastructure: Best practices to reduce environmental impact without compromising efficiency or budget.

> ## More Than an Event, an Experience

Reviewer: ```suggestion
## More than another event, this was a fully immersed experience
```

> KCD Mexico 2025 was more than just a conference; it was a community-building experience. Attendees not only learned

Reviewer: ```suggestion
KCD Mexico 2025 was more than just another conference; it was a community-building experience. Attendees not only learned
```

from global leaders but also shared experiences, challenges, and lessons with peers from across Latin America.
The Q&A sessions sparked in-depth discussions about implementation, common pitfalls, and innovative production solutions,
all in a collaborative, open, and energetic environment.

## Community + Technology = Future

> The event reaffirmed the community's vital role in the growth of the Kubernetes ecosystem. Topics such as DevOps,
> CI/CD, Crossplane, EKS, Istio, and KubeVirt were presented as concrete tools to tackle current infrastructure challenges.
> This event not only strengthened the local community but also equipped each participant with ideas, connections,
> and skills ready to be put into practice.

Reviewer: ```suggestion
KCD Mexico was a vibrant celebration of collaboration, innovation, and technical excellence. Set in Guadalajara—widely recognized as the Silicon Valley of Mexico—the event underscored the city's growing influence as a hub for cloud-native and open-source technologies. It reaffirmed the community's pivotal role in driving the Kubernetes ecosystem forward.

From DevOps and CI/CD to Crossplane, EKS, Istio, and KubeVirt, the sessions showcased powerful tools and forward-thinking strategies to address today’s infrastructure challenges. More than just a conference, KCD Mexico served as a catalyst—connecting professionals, sparking new ideas, and empowering participants with practical knowledge to implement immediately.
```
